playbooks/roles/ocp-customization/tasks/powervm_rmc.yaml

```diff
@@ -1,4 +1,12 @@
+- name: Insure able to fetch project list
+  shell: "oc get projects"
+  register: projects_list
+  until: projects_list.stdout.find("default") != -1
+  retries: 60
+  delay: 5
```

Comment:
@christopher-horn changes lgtm but can we just change the delay to `20`s? Too many RETRYING messages on console does not look good.
Also, retries can be set to `15` (total 5 mins) or `30` (total 10 mins).

Reply:
Certainly, let's make it 30 s on delay with 20 retries.  For normal deploys shouldn't ever need to wait anyway.